js/components/discussions/threads-form.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+<form role="form" class="clearfix animated" v-on:submit.prevent="submit">
+    <div class="form-group">
+        <label for="title-new-discussion">{{ _('Title') }}</label>
+        <input type="text" id="title-new-discussion" v-model="title" class="form-control" required />
+        <label for="comment-new-discussion">{{ _('Comment') }}</label>
+        <textarea id="comment-new-discussion" v-model="comment" class="form-control" rows="3" required></textarea>
+    </div>
+    <button type="submit" class="btn btn-primary btn-block pull-right submit-new-discussion">
+        {{ _('Start a discussion') }}
+    </button>
+</form>
+</template>
+
+<script>
+import i18n from 'i18n';
+import Auth from 'auth';
+import log from 'logger';
+
+export default {
+    props: {
+      subjectId: String,
+      subjectClass: String
+    },
+    methods: {
+        submit(event) {
+            if (!Auth.need_user(i18n._('You need to be logged in to comment.'))) {
+                return;
+            }
+            const data = {
+                title: this.title,
+                comment: this.comment,
```

Comment:
Doesn't [this](https://github.com/opendatateam/udata/pull/520/files#diff-59f9ba3619a1ac796f86368f4030b466R9) do the job?